scrapers/urbandictionary.rb

```diff
@@ -21,6 +25,13 @@ def get_definitions(term)
     definitions.map { |m| parse_definition(term, m) }
   end
 
+  def get_random_definition()
+    uri = URL(DOMAIN+RANDOM)
```

Comment:
Sounds good! I think `URI` and `URL` are close enough in implementation, but we should probably use `URI`?